sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -68,27 +68,22 @@ function getTopicFilter(value: any): SqlRuleFilter | CorrelationRuleFilter {
 /**
  * @internal
  * @ignore
- * Helper utility to retrieve rule `action` value from given input,
- * or undefined if not passed in.
+ * Helper utility to retrieve rule `action` value from given input.
  * @param value
  */
-function getRuleActionOrUndefined(value: any): SqlRuleAction | undefined {
-  if (value == undefined) {
-    return undefined;
-  } else {
-    return {
-      sqlExpression: value["SqlExpression"],
-      sqlParameters: getSqlParametersOrUndefined(value["Parameters"]),
-      compatibilityLevel: getIntegerOrUndefined(value["CompatibilityLevel"]),
-      requiresPreprocessing: getBooleanOrUndefined(value["RequiresPreprocessing"])
-    };
-  }
+function getRuleAction(value: any): SqlRuleAction {
+  return {
+    sqlExpression: value["SqlExpression"],
+    sqlParameters: getSqlParametersOrUndefined(value["Parameters"]),
+    compatibilityLevel: getIntegerOrUndefined(value["CompatibilityLevel"]),
+    requiresPreprocessing: getBooleanOrUndefined(value["RequiresPreprocessing"])
+  };
 }
 
 /**
- * Represents all attributes of a rule entity
+ * Represents the options to create a rule for a subscription.
  */
-export interface RuleProperties {
+export interface CreateRuleOptions {
```

Comment:
This interface can be deleted now right?

Reply:
Ah, looks like we use this internally. So it should need `@internal` and `@ignore` flags

Reply:
Yes

Reply:
Done!